src/cur-mot/common/html_2014_2025.py

```diff
@@ -0,0 +1,552 @@
+from common.html_common import (
```

Comment:
Such a long script definitely needs a description! Preferably a long and detailed one so that we know what's going on.

Reply:
Comments would be good, too.